File: go/utility/feature_add.go
package utility

import "github.com/voxgig-sdk/no-as-a-service-two-sdk/go/core"

func featureAddUtil(ctx *core.Context, f core.Feature) {
	client := ctx.Client
	features := client.Features

	client.Features = append(features, f)
}

File: php/utility/FeatureAdd.php
<?php
declare(strict_types=1);

// NoAsAServiceTwo SDK utility: feature_add

class NoAsAServiceTwoFeatureAdd
{
    public static function call(NoAsAServiceTwoContext $ctx, mixed $f): void
    {
        $ctx->client->features[] = $f;
    }
}
